src/app/components/LinkedData/index.tsx

```diff
@@ -5,7 +5,28 @@ import serialiseForScript from '#lib/utilities/serialiseForScript';
 import getBrandedImage from '#lib/utilities/getBrandedImage';
 import { ServiceContext } from '../../contexts/ServiceContext';
 import getAboutTagsContent from './getAboutTagsContent';
-import { LinkedDataProps } from './types';
+import { BylineLinkedData, LinkedDataProps } from './types';
+
+type OrgAuthor = {
+  noBylinesPolicy?: string | null | undefined;
+  '@type': string;
+  name: string;
+  logo: {
+    '@type': string;
+    width: number;
+    height: number;
+    url: string;
+  };
+};
+
+type AuthorStructure = {
+  image?: string | undefined;
```

Comment:
I don't think you need to include undefined if the field is optional.
```suggestion
  image?: string;
```